spec/test_views.yml

```diff
@@ -0,0 +1,17 @@
+categories:
```

Comment:
I think this test configuration and custom view should be put on example or fixture directory.
Putting similar test file on same directory is not developer friendly.

Reply:
I think so, so I moved views option test to fixture directory.